presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +using a hash of the join key. This can be slower       +
++                                          +                                          +                                    +(sometimes substantially) than broadcast joins but     +
++                                          +                                          +                                    +allows much larger joins. Broadcast joins require that +
++                                          +                                          +                                    +the tables on the right side of the join fit in memory +
++                                          +                                          +                                    +on each machine whereas distributed joins only need to +
++                                          +                                          +                                    +fit in distributed memory across all machines. This    +
++                                          +                                          +                                    +can also be specified on a per-query basis using the   +
++                                          +                                          +                                    +``distributed_join`` session property.                 +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++   ``node-scheduler.network-topology``    +   ``String`` (``legacy`` or ``flat``)    +             ``legacy``             +Sets the network topology to use when scheduling       +
++                                          +                                          +                                    +splits. ``legacy`` will ignore the topology when       +
++                                          +                                          +                                    +scheduling splits. ``flat`` will try to schedule       +
++                                          +                                          +                                    +splits on the same host as the data is located by      +
++                                          +                                          +                                    +reserving 50% of the work queue for local splits.      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.default-concurrency``       +               ``Integer``                +               ``1``                +Default local concurrency for parallel operators. Use  +
```

Comment:
I don't know what findings are you referring to, but I think this as only default for many things is hard to describe in general. I think we can write more about ``task_hash_build_concurrency``, ``task_aggregation_concurency`` and ``task.join_concurrency`` (``task_join_concurrency``) in order to give some kind of advice about concurrency.

As for general value the only thing that comes to my mind is that we may advice that to be higher in low traffic (low number of queries) cluster with multicore nodes and large queries in order to utilize cluster hardware.